caddytls/certificates.go

```diff
@@ -182,6 +182,23 @@ func makeCertificate(certPEMBlock, keyPEMBlock []byte) (Certificate, error) {
 	if err != nil {
 		log.Printf("[WARNING] Stapling OCSP: %v", err)
 	}
+	// TODO: We don't have the Config.CTLogURLs here, just hardcoded some
+	// Config.Managed (with Let's Encrypt) specific values here.
```

Comment:
Easy enough I think: `makeCertificate` is called only one place where Managed certificates are loaded: `CacheManagedCertificate` which does have access to a Config. Maybe the list can be passed in as an argument? And for the other two places where this function is called, it's just one more function up. We can start with that but if it gets too messy then we can look at relocating this particular part of the code.